package.json

```diff
@@ -84,6 +84,7 @@
     "arrowParens": "avoid"
   },
   "files": [
+    "build/**",
     "src/**"
```

Comment:
Sounds good to me, actually, I think we can also:
- Drop the `src` folder from the artifact
- Add `declaration: true` to `tsconfig.json` so that the `d.ts` files are in `build` directory

Reply:
I would argue we don't need the `src/**/*.ts` files. Otherwise, I  agree to include the build files. Probably I'm wrong, but I would either use .npm or the `files` field, not sure we need to set both.

Reply:
> but I would either use .npmignore or the files field, not sure we need to set both.

you need both, without the`"files": [` in the package.json, it will not bundle them, and `.npmignore` it will default to `.gitignore` and will just ignore the build directory.

Reply:
I've just tried and `files: [ ]` has higher priority than .gitignore so if we:
- set `files: [ "build/**" ]`
- delete `.npmignore`

we should be good (unless I tried wrong)